### ISSUE TITLE
[WPF] Check for design mode check must be done on UI thread

### ISSUE THREAD

**Issue by mark-monteiro** [beto-rodriguez/LiveCharts2#164]
**Describe the bug**
Loading a cartesian chart in a WPF application produces the following exception. I have confirmed this still happens in my application when using a totally bare chart with no data/axis/etc. applied, though I was having trouble putting together a minimal reproducible example.

```
System.InvalidOperationException: The calling thread cannot access this object because a different thread owns it.
   at System.Windows.Threading.Dispatcher.VerifyAccess()
   at System.Windows.DependencyObject.GetValue(DependencyProperty dp)
   at System.ComponentModel.DesignerProperties.GetIsInDesignMode(DependencyObject element)
   at LiveChartsCore.SkiaSharpView.WPF.Chart.get_DesignerMode()
   at LiveChartsCore.Chart`1.UpdateThrottlerUnlocked()
   at LiveChartsCore.Kernel.ActionThrottler.Call()
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__140_1(Object state)
   at System.Threading.QueueUserWorkItemCallback.<>c.<.cctor>b__6_0(QueueUserWorkItemCallback quwi)
   at System.Threading.ExecutionContext.RunForThreadPoolUnsafe[TState](ExecutionContext executionContext, Action`1 callback, TState& state)
   at System.Threading.QueueUserWorkItemCallback.Execute()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
```

**To Reproduce**
I found it difficult to reproduce this error. Perhaps because it only happens under specific async or multi-threaded behavior. But in my application, adding any `CartesianChart` to a view causes this exception.

**Expected behavior**
No exception. `LiveChartsCore.SkiaSharpView.WPF.Chart.DesignerMode` should either be altered to work on any thread, or only be called from the UI thread.

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: Beta-60 (I have also confirmed this issue does NOT exist in Beta-50)

**Possible Solutions**
The current implementation to get the design-mode state for WPF calls `DesignerProperties.GetIsInDesignMode(this)`, where `this` is the WPF `Chart` control. The chart control is created on the UI thread and can therefore only be accessed on the UI thread.

A possible workaround would be to make the implementation independent of the calling thread by changing it to `DesignerProperties.GetIsInDesignMode(new DependencyObject());` (taken from here: https://stackoverflow.com/a/834324/1988326). The downside of this solution is that is requires instantiating a new DepdendencyObject every time this property is called, which isn't great.

Alternatively, you could make sure to always access the `DesignerMode` property from the UI thread. In this specific case that would mean updating the [`UpdateThrottlerUnlocked()`](https://github.com/beto-rodriguez/LiveCharts2/blob/master/src/LiveChartsCore/Chart.cs#L412) method to move the call to `View.DesignerMode` inside the `View.InvokeOnUIThread` block

**Comment by beto-rodriguez** [comment]
That is a great report, thanks.

**Comment by beto-rodriguez** [comment]
Fixed with 3b41d0f1beac27ffd51a9e9eeb5ebc7cdf1eeb6f thanks for the report, I disabled any update attempt if the chart is in designer mode.
